store/src/main/java/org/apache/rocketmq/store/DefaultMessageStore.java

```diff
@@ -1552,26 +1552,28 @@ public void run2() {
             }
         }, 1, 10, TimeUnit.MINUTES);
 
-        this.scheduledExecutorService.scheduleAtFixedRate(new AbstractBrokerRunnable(this.getBrokerIdentity()) {
-            @Override
-            public void run2() {
-                if (DefaultMessageStore.this.getMessageStoreConfig().isDebugLockEnable()) {
+        if (DefaultMessageStore.this.getMessageStoreConfig().isDebugLockEnable()) {
```

Comment:
I think this is  better  to put in the `run()`, can support the debugLockEnable dynamic modification, WDYT

Please let me know if I am missing something

Reply:
Thanks, It can indeed be dynamically updated